client/blocks/signup-form/social.jsx

```diff
@@ -40,6 +42,7 @@ class SocialSignupForm extends Component {
 		const extraUserData = response.user && {
 			user_name: response.user.name,
 			user_email: response.user.email,
+			is_dev_account: this.props.isDevAccount,
```

Comment:
This will only be called if `response.user` is defined. We should dispatch it even if the user was not returned from authentication:

```suggestion
		const extraUserData = {
			is_dev_account: this.props.isDevAccount,
			...(response.user && {
				user_name: response.user.name,
				user_email: response.user.email
			} )
		}
```